onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -215,6 +217,11 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
                     return
             else:
                 return
+        elif normalize_node.op_type == 'LayerNormalization':
```

Comment:
It's better to add a test if this is to fuse attention for a new model

Reply:
Actually, this is to support pre-layer normalization. The original bert has normalization after attention and ffn. But, some architectures have better training with normalization before attention and ffn. It's introduced in this PR (https://github.com/microsoft/onnxruntime/commit/a028ca41ecc1df40b49b6fa8bdf11287816bea16), but it misses one case.